map/C:\Users\isaacgateno\Downloads\ANES_Issues_2020.csv

```diff
@@ -0,0 +1,57 @@
+"Issue","Cor_to_Party_ID","Normalized_Variance","Type","URL"
```

Comment:
The filename here is a little weird, with the `C:\Users...` path - can you simplify that?